pkg/controllers/updaterun/execution.go

```diff
@@ -123,7 +133,9 @@ func (r *Reconciler) executeUpdatingStage(
 			finishedClusterCount++
 			continue
 		}
-		// The cluster is either updating or not started yet.
+		clusterUpdatingCount++
```

Comment:
Not able to comment above so leaving the comment here: your line 128 error can also be aggregated instead of directly returned? In this case we are able to report the other cluster's status even if some of them fail.